### PR TITLE
Fixes formatting of localized strings.

### PR DESCRIPTION
**Pull request by omaciel**
This should fix several CLI issues that were filed with latest build,
including BZ #1007128.

### CODE REVIEW

File: src/katello/client/core/base.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def _get_usage_line(self, command_name, parent_usage):
         first_line += " "
         first_line += command_name or ""
         if self.takes_options:
-            first_line += " <{0}>".format(_("options"))
+            first_line += " %s" % _("options")
```
Comment (komidore64) [inline src/katello/client/core/base.py:112]: moving back to old-style string formatting? seems odd...

**Comment by komidore64** [comment]
there are other places in the cli that use `str.format()`. do those need to be changed also?

**Comment by omaciel** [comment]
I can take a look at do some more tweaks.

**Comment by komidore64** [comment]
plox2squash?

commit message should be:
`<BZ#> - <meaningful message>`

`mend` isn't exactly descriptive :wink:

**Comment by komidore64** [comment]
ty `ack`

**Pull request merged**